src/settings_gui.cpp

```diff
@@ -215,6 +265,13 @@ struct GameOptionsWindow : Window {
 				}
 				break;
 
+			case WID_GO_REFRESH_RATE_DROPDOWN: // Setup refresh rate dropdown
+				for (uint i = 0; i < refresh_rates.size(); i++) {
```

Comment:
Sure there is a better C++ construct here .. `for (auto refresh_rate : refresh_rates)`? (I don't know, I am guessing .. new to C++)

Reply:
Good suggestion, but I need to know the index of the refresh rate for `DropDownListCharStringItem`.